### ISSUE TITLE
Python-pip error 

### ISSUE THREAD

**Issue by ShadowBeast235** [abhi-r3v0/Adhrit#46]
I
![Screenshot_2021-02-04_02_15_00](https://user-images.githubusercontent.com/26254355/106857913-dcc1a700-668e-11eb-98fd-44c4b32d7383.png)
t requires python-pip but linux latest build dosnt have python-pip beacuse python2 is dead. 
In discription you mention this tool worked on linux/Mac still i cant run this framework help me to resolve this issue

**Comment by abhi-r3v0** [comment]
Hi, 

Was keeping pip for backwards compatibility. Will push an update for the same.
Alternatively, you can install the pip3 tools manually from `installer.py`.

Regards

**Comment by slindy** [comment]
Hi

what is the solution here plz ?

![image](https://user-images.githubusercontent.com/3879402/132092191-5930f41b-b304-48cf-99b1-e740d192dda8.png)

**Comment by abhi-r3v0** [comment]
Hi,

You can open the `config` file and set `dependencies_status` = complete and try again. If anything fails, let us know.

Thanks

**Comment by slindy** [comment]
Hello

Thanks for your response.
On windows doesn't work.
On Kali doesn't work
On ubuntu i got this error

![image](https://user-images.githubusercontent.com/3879402/132229280-fff9ac36-5812-43c8-8803-88a2e806e201.png)

**Comment by slindy** [comment]
Hello

On kalli

![image](https://user-images.githubusercontent.com/3879402/132242233-7f246f9d-e611-4d23-b677-460ec4e64fb9.png)

**Comment by kali64amd** [comment]
yup! its not working after changing config file status complete.

Any Input guys???
